ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.14.yaml

```diff
@@ -0,0 +1,163 @@
+base_images:
+  manila-csi-driver-operator-test:
+    name: "4.14"
+    namespace: ocp
+    tag: manila-csi-driver-operator-test
+  openstack-cinder-csi-driver-operator-test:
+    name: "4.14"
+    namespace: ocp
+    tag: openstack-cinder-csi-driver-operator-test
+  openstack-installer:
+    name: "4.14"
+    namespace: ocp
+    tag: openstack-installer
+  openstack-tests:
+    name: "4.14"
+    namespace: ocp
+    tag: openstack-tests
+releases:
+  initial:
+    candidate:
+      product: ocp
+      relative: 1
+      stream: ci
+      version: "4.14"
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-openstack-fips
+  minimum_interval: 48h
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      FIPS_ENABLED: "true"
+      TEST_SUITE: experimental/reliability/minimal
+    pre:
+    - chain: ipi-openstack-pre
+    - ref: fips-check
+    workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-proxy
+  minimum_interval: 72h
+  steps:
+    cluster_profile: openstack-vh-mecha-central
+    workflow: openshift-e2e-openstack-proxy
+- as: e2e-openstack-externallb-techpreview
+  minimum_interval: 168h
+  steps:
+    cluster_profile: openstack-vh-mecha-central
+    workflow: openshift-e2e-openstack-external-lb
+- as: e2e-openstack-nfv-intel
+  minimum_interval: 72h
+  steps:
+    cluster_profile: openstack-nfv
+    workflow: openshift-e2e-openstack-nfv
+- as: e2e-openstack-nfv-intel-kuryr
+  minimum_interval: 72h
+  steps:
+    cluster_profile: openstack-nfv
+    env:
+      NETWORK_TYPE: Kuryr
+    workflow: openshift-e2e-openstack-nfv
+- as: e2e-openstack-nfv-mellanox
+  minimum_interval: 72h
+  steps:
+    cluster_profile: openstack-nfv
+    env:
+      ADDITIONAL_WORKERS_NETWORKS: mellanox-dpdk
+      OPENSTACK_DPDK_NETWORK: mellanox-dpdk
+      OPENSTACK_SRIOV_NETWORK: mellanox-sriov
+    workflow: openshift-e2e-openstack-nfv
+- as: e2e-openstack-additional-ipv6-network
+  minimum_interval: 72h
+  steps:
+    cluster_profile: openstack-hwoffload
+    workflow: openshift-e2e-openstack-ipv6
+- as: e2e-openstack-csi-manila
+  minimum_interval: 72h
+  steps:
+    cluster_profile: openstack-vh-mecha-central
+    workflow: openshift-e2e-openstack-csi-manila
+- as: e2e-openstack-az
+  minimum_interval: 48h
+  steps:
+    cluster_profile: openstack-vh-mecha-az0
+    env:
+      ZONES_COUNT: "1"
+    workflow: openshift-e2e-openstack-proxy
+- as: e2e-openstack-csi-cinder
+  minimum_interval: 72h
+  steps:
+    cluster_profile: openstack-vh-mecha-central
```

Comment:
Can you also add a skip of the tests listed here https://github.com/openshift/release/pull/37653?

Thanks

Reply:
good point, done